### DISCUSSION TITLE
DSharpPlus.CommandsNext.BaseCommandModule not found

### DISCUSSION THREAD

**Discussion by Einhornyordle** [DSharpPlus/DSharpPlus#779] [Support] [answered]
# Summary
As I said, I tried to create a Command Module by following the instructions over at [the documentation](https://dsharpplus.github.io/articles/commands/intro.html#create-a-command-module ) but when I try to inherit from BaseCommandModule it "could not be found".

# Details
My Code:
```
using DSharpPlus.CommandsNext;
using System.Threading.Tasks;

namespace TheCrystalBot
{
    public class CommandModule : BaseCommandModule
    {

    }
}
```
Build output:
```
1>------ Build started: Project: TheCrystalBot, Configuration: Debug Any CPU ------
1>D:\dev\c#\TheCrystalBot\TheCrystalBot\Commands\CommandModule.cs(6,34,6,51): error CS0246: The type or namespace name 'BaseCommandModule' could not be found (are you missing a using directive or an assembly reference?)
1>Done building project "TheCrystalBot.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```
My Setup:
IDE:  VS 2019 Community
OS: Windows 10
.NET Core 3.1
NuGet Packages: DSharpPlus, DSharpPlus.CommandsNext, DSharpPlus.Interactivity, DSharpPlus.VoiceNext (all of them are v3.2.3)

# Notes
I tired rebuilding the project after cleaning the solution, removing and readding the DSharpPlus.CommandsNext NuGet Package and restarting VS, however, nothing worked. The only Module classes I could find are BaseModule and CommandsNextModule but no Base​Command​Module.

Edit: Screenshot
![image](https://user-images.githubusercontent.com/47315090/108998616-77674180-76a1-11eb-83b2-cc0fbdd89b08.png)

**Comment by Emzi0767** [comment, marked as answer]
In the future, use discussion forums. Furthermore, you're reading 4.x docs and trying to apply them to 3.x. Upgrade.
